setup.py

```diff
@@ -15,35 +15,42 @@
 readme = io.open('README.md', 'r', encoding='utf-8').read()
 
 setup(
-    name='storyscript-platform-engine',
-    description='The engine of the Storyscript platform',
+    name='storyruntime',
```

Comment:
```suggestion
    name='story-runtime',
```

https://pypi.org/project/story-runtime